### PR TITLE
add title to swatches in interpolatedColorLegend

### LINKED ISSUE #2887: [InterpolatedColorLegend] Add <title>s to swatches
On `Legend`, there are `<title>` tags on the text so the text is still readable even if it gets cut off. We should add `<title>` tags to the swatches on `InterpolatedColorLegend` so the values associated with each swatch can be read.

Note that this should take `Configs.ADD_TITLE_ELEMENTS` into account.

### PR DESCRIPTION
**Pull request by ztsai**
add title to each swatch in `interpolatedColorLegend` if `Configs.ADD_TITLE_ELEMENTS` is true
demo: http://jsfiddle.net/ztsai/r3ozqxre/ http://jsfiddle.net/ztsai/r3ozqxre/3/
closes #2887

### CODE REVIEW

**Comment by jtlan** [comment]
Can we use the `Formatter` on the `InterpolatedColorLegend` to format the `<title>`s as well?

File: plottable.css [hunk 1]
```diff
@@ -211,6 +211,7 @@ svg.plottable {
   fill: none;
   stroke: #CCC;
   stroke-width: 1px;
+  pointer-events: visiblestroke;
```
Comment (jtlan) [inline plottable.css]: Why is this change required?

Comment (ztsai) [inline plottable.css]: `.swatch-bounding-box` is drawn on top of swatches for outline, and without this css attribute, hover won't toggle tooltip

Comment (jtlan) [inline plottable.css]: Wouldn't `pointer-events: none;` be more semantically correct then?

**Comment by crmorford** [comment]
tested on erry browser but not mobile because <title> doesn't work on mobile
